packages/turbo-gen/src/commands/generate/prompts.ts

```diff
@@ -8,24 +9,26 @@ export async function customGenerators({
   generators: Array<Generator | inquirer.Separator>;
   generator?: string;
 }) {
-  if (
-    generator &&
-    generators.find(
-      (g) => !(g instanceof inquirer.Separator) && g.name === generator
-    )
-  ) {
-    return {
-      selectedGenerator: generator,
-    };
+  if (generator) {
+    if (
+      generators.find(
+        (g) => !(g instanceof inquirer.Separator) && g.name === generator
+      )
+    ) {
+      return {
+        selectedGenerator: generator,
+      };
+    }
+
+    logger.warn(`Generator "${generator}" not found`);
```

Comment:
If you passed a generator name that is invalid, we warn here, and then send you to the picker.